docs/adr/adr_statviz_fe_sharing.md

```diff
@@ -0,0 +1,148 @@
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
+
+#### comments
+
+- Atm the filtering and grouping is happening on the FE side. Combined with svg previews that is a cool thing. The user can filter and group dynamically without sending requests to the BE. This means that it goes much faster for the user to find the right filter/grouping options that you want for the export of the svg.
+
+- When importing the svg into an email campaign, the svg can be responsive (through CSS) and even basic tooltips and animations (svg SMIL features) should be possible. However, not all email clients support this and probably a fallback bitmap image should be added. At the same time, I'm not sure how well the email editor in mailchimp works and probably you need to add the svg through the code editor of mailchimp.
+
+### C. bitmap (aka PNG)
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. same steps as for svg.
+2. There should be a function to convert the preview svg that is in the fancy graphs view to a png.
+3. (Maybe some some more styling controls like resolution)
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
+
+#### comments
+
+- I prefer png to jpeg since the background can be transparent if they want it.
+
+### D. STATIC iframe embed directly generating html docs for the iframe
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. same steps as for SVG.
+2. There needs to be a function transforming the data from the graphQL query into the iframe embed. The embed itself is probably just an html file, which holds some static data needed for the visualzation, some js to generate the svg based on the data and maybe even some filters and grouping selection.
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
+
+#### comments
+
+Most likely the embed will get large since data and js (maybe even React code) must be in it. I cannot really estimate how much work this might be.
+
+### E. STATIC iframe embed through server-less FE
+
+#### What (minimal) work needs to be done on the FE side?
+
+1. same steps as for SVG.
+2. We create a separate public React FE which is hosted on some Google Bucket and whose route is used for the embed.
+3. We make a shared components out of the svg generation and probably filtering between the Boxtribute v2 and the Statviz public FE.
+4. We create a function that stores the data that should be used for the embedable iframe somewhere in the same Google Bucket where the FE is hosted. (similar approach like forestry for our landing page)
+5. The embed is then just a link to the public statviz frontend and the data that is used for the visualization is also statically stored somewhere in the FE.
+
+#### What (minimal) work needs to be done on the BE side?
+
+1. same steps as for CSV
```

Comment:
I think we need some backend endpoints
1 secured endpoint for creating a database record for the published visualization
1 secured endpoint for getting a list of the published visualizations 
1 secured endpoint for deleting a published visualizations

This would mean that maybe we would need or should have an extra database table to keep track of created/published visualizations

Reply:
I guess these are inside of the minimum requirements, if we value that we want to give the organizations control over their data.